README.md

```diff
@@ -15,7 +15,7 @@ pip install ipython
 ```
 
 ### Step 2: Download Necessary Corpora and Model
-To aid with stopword removal and tokenization, you need to download the required corpora and model. 
+You need to download the required corpora and model to aid with stopword removal and tokenization. 
```

Comment:
You need to download the required corpora that contains the particular language words and model to aid with stopword removal and tokenization.